### DISCUSSION TITLE
Arrow/pyarrow as a backend?

### DISCUSSION THREAD

**Discussion by jiri-kulik** [ibis-project/ibis#3684] [General]
Hi,

just started to explore Ibis. It's a wonderful work! One thing striked me very quickly though - missing Arrow backend. Is it just nobody has time to work on it or is it a design decision?

**Comment by gforsyth** [comment]
Hey @jiri-kulik -- glad you are enjoying your Ibis exploration!  We definitely have an Arrow backend on our roadmap (we also have publishing a roadmap on our roadmap...), so stay tuned!

**Comment by cpcloud** [comment]
This is an interesting idea, but I don't think we're likely to take this on soon.

We do however have a number of backends that work natively with pyarrow objects (tables, record batch readers, etc), so you should be 100% unblocked on in-memory analytics with those objects.
